### ISSUE TITLE
Generic relationship object_id fields should use PositiveIntegerField

### ISSUE THREAD

**Issue by osamak** [django-notifications/django-notifications#199]
Currently, `actor_object_id`, `target_object_id`, `action_object_object_id` are all `CharField` instead of `PositiveIntegerField`.

This is problematic at least in Django set-ups that use PostgreSQL, which fail to run queries involving primary keys (which are integer fields) and django-notification's `object_id` fields  a  with the following error message:

```
django.db.utils.ProgrammingError: operator does not exist: integer = character varying
LINE 1: ...M "app_model" WHERE NOT ("app_model"."id" IN (SELECT...
                                                             ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
```

*(where app_model is the actual database table)*

Please change object_id fields from `CharField` to `PositiveIntegerField`

**Comment by osamak** [comment]
This one is related to #125

**Comment by osamak** [comment]
If for any reason this cannot be fixed, a solution should be posted into `README.md`

**Comment by nemesisdesign** [comment]
@osamak it cannot be an integer field because some people use UUID fields in their applications, which are alphanumeric strings. Please close this issue, if you think it should be added to the docs, send a PR.

**Comment by udos** [comment]
sorry about this. also in my opinion this issue (together with #125) should be closed (or marked as duplicate of #125). a fix or workaround is described in comment https://github.com/django-notifications/django-notifications/issues/125#issuecomment-371756082.
